trade-finance-manager-ui/server/controllers/utilisation-reports/confirm-delete-payment/index.ts

```diff
@@ -28,16 +30,23 @@ const getPaymentSummaryListRows = (paymentResponse: GetPaymentDetailsWithoutFeeR
   ];
 };
 
-export const getConfirmDeletePayment = async (req: Request, res: Response) => {
+type GetConfirmDeletePaymentRequest = CustomExpressRequest<{
+  query: {
+    redirectTab?: ReconciliationForReportTab;
```

Comment:
Is there only 1x tab in the page?

If not, i'm wondering if this should have a more specific name e.g `redirectXTab` - just thinking, if a new tab redirect is added we'd have to rename this

Reply:
There are multiple tabs on the reconciliation for report page and this query param specifies which of these we want to be shown when we perform a redirect. This page is the only template within the utilisation reports group that uses tabs at present. We also need to be aware when naming that this is passed around as a query parameter.

We could rename it something like `reconciliationForReportRedirectTab` if you'd prefer something like that? Or leave as is? Open to either, thanks

Reply:
Thanks for explaining @sw-mattw, I think what we have already makes sense 👍